src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-tabs/cephfs-tabs.component.html

```diff
@@ -1,31 +1,30 @@
 <ng-container *ngIf="selection">
-  <nav ngbNav
-       #nav="ngbNav"
-       (navChange)="softRefresh()"
-       class="nav-tabs"
-       cdStatefulTab="cephfs-tabs">
+  <nav
+    ngbNav
+    #nav="ngbNav"
+    (navChange)="softRefresh()"
+    class="nav-tabs"
+    cdStatefulTab="cephfs-tabs"
+  >
```

Comment:
is it possible that you have an html-linter configured in your IDE? because these changes looks like just changing the formatting. You can run `npm run lint:html` from the frontend folder to see if this gets caught in any error

Reply:
ah ok. I'll amend my IDE html formatter bc I think it is changing things automatically. Fixed!